cmd/kubestellar-where-resolver/cmd/where-resolver-cmd.go

```diff
@@ -104,13 +106,20 @@ func Run(ctx context.Context, options *resolveroptions.Options) error {
 		logger.Error(err, "failed to create edge clientset for controller")
 		return err
 	}
+	serviceProviderClient, err := kubernetes.NewForConfig(espwRestConfig)
```

Comment:
I would not use the words "service provider" in this variable name. That only identifies its significance in the theory of kube-bind and `kbuser.NewKubeBindSpaceRelation`. It would be better if the variable name more specifically identified _which_ service provider it is talking about. That would be the KCS (née ESPW). Even better, since this is not the only clientset for that space, would be for the variable to identify which API group(s) _this_ clientset is for. Perhaps something like `kubeClient` (and perhaps change `espwClientset` above to `edgeClient`).

Reply:
Changed